experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_k8s_job.yml

```diff
@@ -0,0 +1,105 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: disk-loss-
+spec:
+  template:
+    metadata:
+      labels:
+        name: disk-loss
+    spec:
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:ci
+        imagePullPolicy: Always
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          # Enter the App_label, if you set APP_CHECK to true
+          - name: APP_LABEL
+            value: 'app=mysql'
```

Comment:
This is the default value?

Reply:
Yes

Reply:
It will remain the same for all experiments? I think it will change on the basis of the AUT.